propagation/b3/http.go

```diff
@@ -31,8 +55,13 @@ func ExtractHTTP(r *http.Request) propagation.Extractor {
 			parentSpanIDHeader = r.Header.Get(ParentSpanID)
 			sampledHeader      = r.Header.Get(Sampled)
 			flagsHeader        = r.Header.Get(Flags)
+			singleHeader       = r.Header.Get(Context)
 		)
 
+		if singleHeader != "" {
+			return ParseSingleHeader(singleHeader)
```

Comment:
only return if successful... if error and we have multi header data we should try that as fallback before we give up